### ISSUE TITLE
Migrate other platform

### ISSUE THREAD

**Issue by suyulin** [ardera/flutter-pi#188]
hi,
I migrated my project to another platform and it reported an error
![image](https://user-images.githubusercontent.com/6275411/125410410-7126f700-e3ef-11eb-94e5-1978582195ee.png)
Looking forward to your reply

**Comment by suyulin** [comment]
egl 1.5 has getPlatformDisplay ,but egl 1.4 should not have this api, why can use?

**Comment by ardera** [comment]
Yes, `eglGetPlatformDisplay` is EGL 1.5 only. In earlier EGL it's called `eglGetPlatformDisplayEXT` instead, I'll change the logic so it falls back to that if the 1.5 version is not found

**Comment by ardera** [comment]
please don't close this, I'll close it when it's resolved instead
